File: src/de/ax/powermode/TogglePowerMode.scala
package de.ax.powermode

import com.intellij.openapi.actionSystem.{AnAction, AnActionEvent}

/**
  * Created by nyxos on 31.03.16.
  */
class TogglePowerMode extends AnAction with Power {
  def actionPerformed(e: AnActionEvent): Unit = {
    powerMode.setEnabled(!powerMode.isEnabled)
  }
}

File: src/com/nmeylan/powermode/TogglePowerMode.java
package com.nmeylan.powermode;

import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;

public class TogglePowerMode extends AnAction implements Power {

  @Override
  public void actionPerformed(AnActionEvent e) {
    powerMode().setEnabled(!powerMode().isEnabled());
  }
}
